packages/cli/test/unit/util/to-host.test.ts

```diff
@@ -1,3 +1,4 @@
+import { describe, expect, it } from 'vitest';
```

Comment:
Just randomly came across this PR, but these imports could be avoided for each file when specifying `globals: true` within `vitest.config.ts` and adding `"vitest/globals"` to `compilerOptions.types` in `tsconfig.json`

Reply:
Thanks for the note, but we have explicit imports on purpose. We prefer seeing exactly where these functions come from.